### DISCUSSION TITLE
Script Not Running on DS920+

### DISCUSSION THREAD

**Discussion by OoogaBooooga** [RandomNinjaAtk/docker-lidarr-extended#38] [Q&A] [answered]
Lidarr extended pulls songs off either tidal or deezer and downloads them right? If so, do I need to put in indexers or downloads clients? I’m assuming not. I’ve done a test Spotify import and got the albums recognized but I don’t see songs getting downloaded. Is there another step I should do? Below is my docker compose file. Is there anything else I should add? Please and Thanks

version: "2.1"
services:
  lidarr-extended:
    image: randomninjaatk/lidarr-extended:latest
    container_name: lidarr-extended
    volumes:
      - /volume1/docker/lidarr-extended:/config
      - /volume1/data/downloads:/downloads-lidarr-extended
      - /volume1/data/downloads/music-videos:/music-videos
      - /volume1/data:/music
    environment:
      - TZ=Canada/Mountain
      - PUID=1027
      - PGID=100
      - autoStart=true
      - enableAudioScript=true
      - scriptInterval=15m
      - configureLidarrWithOptimalSettings=true
      - searchSort=date
      - audioFormat=native
      - audioBitrate=lossless
      - requireQuality=true
      - enableBeetsTagging=true
      - enableReplaygainTags=true
      - audioLyricType=both
      - dlClientSource=both
      - arlToken=e83.............673
      - tidalCountryCode=US
      - addDeezerTopArtists=true
      - addDeezerTopAlbumArtists=true
      - addDeezerTopTrackArtists=true
      - topLimit=10
      - addRelatedArtists=false
      - numberOfRelatedArtistsToAddPerArtist=5
      - plexUrl=http://x.x.x.x:32400
      - plexToken=Token_Goes_Here
      - youtubeSubtitleLanguage=en
    ports:
      - 8686:8686
    restart: unless-stopped

**Comment by RandomNinjaAtk** [comment]
Check the logs, it’s most likely running but failing because you have to manually authenticate one of the clients. Logs are viewable in the lidarr web interface where it shows the log files or watch the live container log after the container first boots for the first 5 min or so…

**Comment by OoogaBooooga** [comment]
![image](https://user-images.githubusercontent.com/39597403/182740832-c47f889b-7102-421c-840e-584f54c97893.png)

Would I need to download all of them?

**Reply by RandomNinjaAtk** [reply]
You can open any of them to see what's going on, but the fact that you have logs means it's running, but you most likely run into an error...

Just click download on the most recent audio or video script and read what it says, and you'll see what's wrong... It's most likely that you haven't authenticated the one client that requires it, since you have it set to use both. So you'll need to watch the docker container log and authenticate the client before it times out. Or just only use the one client that doesn't require manual authentication...

**Reply by OoogaBooooga** [reply]
Thank you!

**Comment by OoogaBooooga** [comment, marked as answer]
Read the logs and **AUTHENTICATE** Tidal
